### PR TITLE
[1.2] CMAKE files for 1.2.x branch.

### PR DESCRIPTION
**Pull request by Karm**
If there is any demand for it, this is how I built mod_cluster 1.2.x on Windows 10, MSVC 19, with JBoss Enterprise Web Server 2.1.x:
- CMAKE 2.8+
- EWS 2.1.1-ish httpd-devel-2.2.26-19.win6.x86_64
- EWS 2.1.1-ish httpd-2.2.26-19.win6.x86_64
- `cmake ../ -G "NMake Makefiles" -DAPR_LIBRARY=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\lib64\libapr-1.lib -DAPR_INCLUDE_DIR=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\include\httpd\ -DAPACHE_INCLUDE_DIR=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\include\httpd\ -DAPRUTIL_LIBRARY=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\lib64\libaprutil-1.lib -DAPRUTIL_INCLUDE_DIR=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\include\httpd\ -DAPACHE_LIBRARY=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\lib64\libhttpd.lib -DPROXY_LIBRARY=C:\Users\karm\WORKSPACE\MOD_CLUSTER\httpd\devel\lib64\mod_proxy.lib`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,4 @@ native/*/autom4te.cache/
 native/*/config.status
 native/*/configure
 
-native/CMakeLists.txt
+native/build
```
Comment (rhusar) [inline .gitignore]: Extra spaces?

File: native/cmake/modules/FindAPACHE.cmake [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#
+#  APACHE_FOUND         - System has APACHE
+#  APACHE_INCLUDE_DIR   - The APACHE include directory
+#
+#  APACHE_LOCATION      - setting this enables search for apache libraries / headers in this location
+
+#
+# Include directories
+#
+find_path(APACHE_INCLUDE_DIR
```
Comment (rhusar) [inline native/cmake/modules/FindAPACHE.cmake]: The other scripts are using ALL CAPS, is this intentional?

Comment (Karm) [inline native/cmake/modules/FindAPACHE.cmake]: Not pertinent. Let's revamp in teh next iteration where these additional modules will likely go away anyway.

**Comment by Karm** [comment]
@rhusar Ready to be merged. Commit message amended, newlines added.

**Comment by rhusar** [comment]
Merging. 

Re: .gitignore - should there be a problem, the first one to hit this will fix it.

**Pull request merged**